commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountServiceImpl.java

```diff
@@ -102,40 +101,41 @@ public CommerceAccount fetchByExternalReferenceCode(
 	public CommerceAccount fetchCommerceAccount(long commerceAccountId)
 		throws PortalException {
 
+		PermissionChecker permissionChecker = getPermissionChecker();
+
+		_commerceAccountModelResourcePermission.check(
```

Comment:
@alecsloan commerceaccount can be null at this point

Reply:
So the final return should be null right? Instead of `commerceAccountLocalService.getCommerceAccount(
			getUserId(), commerceAccountId)`